Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.Util.cpp

```diff
@@ -12,27 +12,40 @@
 
 int CLuaFunctionDefs::GetValidPedModels(lua_State* luaVM)
 {
-    int iIndex = 0;
+    bool includeCustom;
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadBool(includeCustom, true);
+
+    auto modelManager = g_pClientGame->GetManager()->GetModelManager();
```

Comment:
`auto*`?

Reply:
Why I just copied this from elsewhere

Reply:
Well, guidelines direct you towards the usage of `auto*`. Though do as you will

Reply:
My bad I missed that part